### PR TITLE
Add CloudNet Template of Issue #153

### PR DESCRIPTION
**Pull request by Zoey2936**
Add CloudNet Template of Issue #153

### CODE REVIEW

File: cloudnet/cloudnet.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+      "files": [
+        "https://cloudnetservice.eu/cloudnet/version/release/${version}/CloudNet.zip"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "unzip CloudNet.zip",
+        "chmod -R 777 ."
```
Comment (LordRalex) [inline cloudnet/cloudnet.json]: 777 on an entire folder is bad
Only files which need some special perm should be handled. I would assume chmod +x start.sh

Also, the start.sh is bad for us. It runs screen, which is not what we want. The command should be ripped out, and added appropriately.

Also add java as a requirement. Refer to Minecraft templates.

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: cloudnet/cloudnet.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    {
+      "commands": [
+        "unzip CloudNet.zip"
+      ],
+      "type": "command"
+    }
+  ],
+  "run": {
+    "stop": "exit",
+    "command": "java -XX:+UseG1GC -XX:MaxGCPauseMillis=50 -XX:+UnlockExperimentalVMOptions -XX:+UseCompressedOops -XX:-UseAdaptiveSizePolicy -XX:CompileThreshold=100 -Dfile.encoding=UTF-8 -Xmx${memory}M -Xms${memory}m -jar launcher.jar",
```
Comment (LordRalex) [inline cloudnet/cloudnet.json]: Doesn't use the variable defined below

**Review by LordRalex — CHANGES_REQUESTED** [review]

**Comment by LordRalex** [comment]
Also missing requirements. Refer to Minecraft templates for examples

**Comment by Zoey2936** [comment]
It seems that the github api has currently problems with the workflows

**Pull request closed without merging**